packages/Table/src/index.js

```diff
@@ -1 +1,2 @@
+export { TH } from "./Table.styles";
```

Comment:
Do we export a styled component in the index of any of our other components?

Would there be a more obvious or scalable API to expose them?

Like, as `<Table.SC.TH>`?  I'm not crazy about that... but actually, maybe just exporting as `<Table.TH>` would be fine – the fact that it's just a _styled_ component doesn't really matter to the consumer.

Reply:
it has to be used, in order to match the border types and even if it's sticky or not. This only affect the edge case of complex header which I think is sane to do it in this way